library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/api/ResourceLoader.java

```diff
@@ -50,6 +51,19 @@ public interface ResourceLoader {
 		return ResourceLoaderImpl.get(type);
 	}
 
+	/**
+	 * Gets the static resource manager for a given type.<br>
+	 * Static resources are different from regular resources in that they
+	 * are available as soon as the manager is requested and are (theoretically) immutable.<br>
+	 * It is <i>highly</i> recommended for mods to personally cache data they get from the provided manager in order to
+	 * prevent changes to userspace packs modifying data at runtime.
+	 * @param type the given resource type
+	 * @return the static resource manager instance
+	 */
+	static @NotNull StaticResourceManager getStaticResourceManager(@NotNull ResourceType type){
```

Comment:
As `StaticResourceManager` is an internal type, this should probably return just a `ResourceManager` to not expose internals

Reply:
Thanks, wouldn't have caught that w.o this comment.